app/controllers/application_controller.rb

```diff
@@ -25,8 +25,9 @@ class ApplicationController < ActionController::Base
     def authorize
       # get user and respective membership
       @user = User.find_by_id(session[:user_id]) 
-      @membership = UserCourseMembership.find_by_user_id(@user.id)
-      
+      if (@user.present?)
```

Comment:
Looks good. Just that the change is already in latest master. It's a good practice to pull the latest version of stable code and apply changes to it.